### DISCUSSION TITLE
DecoRouter

### DISCUSSION THREAD

**Discussion by MrPigss** [encode/starlette#1539] [Ideas]
[DecoRouter ](https://github.com/MrPigss/DecoRouter) is an extension to Starlette that let's you use decorators to generate routing tables.  
This functionality used to be a part of  Starlette but was deprecated and removed from documentation in 0.13.0.

Using DecoRouter can make it easier to migrate from things like Flask and FastApi. There are other pros to using decorators as described in the readme.

Check it out and let me know what you think.

**Comment by lovelydinosaur** [comment]
Saw your post on reddit about this - fantastic stuff. 🌟
Wanna submit a pull request adding it to the third party docs?

**Reply by MrPigss** [reply]
Yes, I saw that it's preferred that a pull request is first mentioned in a discussion, so that's what I did.  
In the meanwhile I get some extra eyes on the project 😄.

**Reply by lovelydinosaur** [reply]
> Yes, I saw that it's preferred that a pull request is first mentioned in a discussion, so that's what I did.

Perfecto. Appreciated. 🙏🏼
